src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -82,22 +118,78 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
         });
     };
 
-    handleFetchSuccess = (responseData: FileVersions, file: BoxItem) => {
+    handleFetchSuccess = ([file, versions]): [BoxItem, FileVersions] => {
         const { api } = this.props;
-        const { entries: versions } = api.getVersionsAPI(false).addCurrentVersion(responseData, file) || {};
+        const { entries } = api.getVersionsAPI(false).addCurrentVersion(versions, file) || {};
```

Comment:
We should integrate the "new" API that allows us to fetch the current version via API at some point

Reply:
Has that been completed and released in v2? I wasn't aware of it if so.

Reply:
It looks like the new change is only applicable to the `/files/:fileId/versions/:versionId` endpoint rather than the list. We can call that endpoint rather than munging in the file object, but it'll take quite a bit more work. Let's look at doing it in a future refactor.